docs/examples/DataSet/Offline plotting with complex data.ipynb

```diff
@@ -98,7 +107,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When visualising the data, `plot_by_id` will turn the complex `signal` parameter into to real parameters. The `plot_by_id` function can do that in one of two ways: either as real and imaginary part or as magnitude and phase. Real and imaginary is the default."
+    "When visualising the data, the `plot_by_id` will turn the complex `signal` parameter into to real parameters. The `plot_by_id` function can do this \"transformation\" in one of two ways: either casting the amplitudes of the real and imaginary parts or calculating the magnitude and phase. By default, the `plot_by_id` uses the former."
```

Comment:
this notebook should all be converted to use `plot_dataset` as opposed to `plot_by_id`
```suggestion
    "When visualising the data, the `plot_dataset` will turn the complex `signal` parameter into to real parameters. The `plot_by_id` function can do this \"transformation\" in one of two ways: either casting the amplitudes of the real and imaginary parts or calculating the magnitude and phase. By default, the `plot_by_id` uses the former."
```